### ISSUE TITLE
Missing nodes in TrustFrameworkPolicy_0.3.0.0.xsd 

### ISSUE THREAD

**Issue by ltouro** [Azure-Samples/active-directory-b2c-custom-policy-starterpack#17]
Following this issue [azure-docs/#6299](https://github.com/MicrosoftDocs/azure-docs/issues/6299#issuecomment-376599298), we noted that  TrustFrameworkPolicy_0.3.0.0.xsd  is missing some nodes.

For instance, this [doc](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/active-directory-b2c/active-directory-b2c-reference-password-complexity-custom.md) mentions InputValidations and Predicates, which are not currently present on the XSD.

The XSD needs to be updated to reflect all available nodes.

**Comment by anielii** [comment]
I encoutered the same issue. Could you please update TrustFrameworkPolicy_0.3.0.0.xsd so that it can be used to validate custom policies?

**Comment by sgdesmet** [comment]
Indeed, TrustFrameworkPolicy_0.3.0.0.xsd misses for example `AlwaysUseDefaultValue` but instead defines `OverwriteIfExists`
